kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue

```diff
@@ -263,7 +263,9 @@
           color: this.$themeTokens.text,
           width: '100%',
           border: '2px solid transparent',
-          'text-align': 'left',
+          'text-transform': 'capitalize',
```

Comment:
Removing this line will be the correct merge conflict resolution and will fix your linting error.
```suggestion
```